cmd/common.go

```diff
@@ -137,11 +141,32 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 			return fmt.Errorf("reading Kong version: %w", err)
 		}
 	}
-	parsedKongVersion, err = parseKongVersion(kongVersion)
+	parsedKongVersion, err = utils.ParseKongVersion(kongVersion)
 	if err != nil {
 		return fmt.Errorf("parsing Kong version: %w", err)
 	}
 
+	if parsedKongVersion.GTE(kongVersion300) &&
```

Comment:
Is there any reason not to use the global utils version? Using it would allow the removal of the local `kongVersion300` variable.

```suggestion
	if parsedKongVersion.GTE(utils.Kong300Version) &&
```

Reply:
No good reason: https://github.com/Kong/deck/pull/753/commits/34fce722c14a34fdc3faae74d75b72e72f22c1eb